src/css/component.site-header.css

```diff
@@ -0,0 +1,107 @@
+/* ---------- COMPONENT site header ---------- */
+
+.site-header {
+  --site-header-gutters-y: var(--size-6);
+  --site-header-menu-gutters-x: var(--size-3);
+  background: var(--color-white);
+  border-bottom: var(--size-px) solid var(--color-grey-500);
+  box-shadow: 0 1rem 0.625rem -0.5rem hsla(var(--color-black-hsl), 0.05);
+  padding-bottom: var(--site-header-gutters-y);
+  padding-top: var(--site-header-gutters-y);
+  position: sticky;
+  top: calc(-1 * var(--size-px));
+  transition: background-color var(--animation-easing) var(--animation-duration-fast),
+    border var(--animation-easing) var(--animation-duration-fast), box-shadow var(--animation-easing) var(--animation-duration-fast);
+}
+
+.site-header__logo {
+  position: relative;
+  width: var(--size-32);
+  z-index: 12;
+}
+
+.site-header__logo svg {
+  height: auto;
+  max-width: 100%;
+  width: 100%;
+}
+
+/* ---------- home ---------- */
+.home .site-header {
+  background: transparent;
+  border: 0;
+  box-shadow: none;
+}
+
+.home .site-header--stuck {
+  background: var(--color-white);
+  border-bottom: var(--size-px) solid var(--color-grey-500);
+  box-shadow: 0 1rem 0.625rem -0.5rem hsla(var(--color-black-hsl), 0.05);
+}
+
+.home .site-header__logo .logo-ceph__text path {
+  fill: #e9eff6; /* TBC */
```

Comment:
Maybe a `transition: fill var(--animation-easing) var(--animation-duration-fast)` here so the logo colour change is as silky smooth as the rest? 😮

Reply:
Agreed.

Also added to the `close` button too.